app/breaksPlanner.js

```diff
@@ -168,34 +168,42 @@ class BreaksPlanner extends EventEmitter {
     this.emit('updateToolTip')
   }
 
-  skipToMicrobreak () {
+  scheduleMicrobreak (delay) {
```

Comment:
I think we should keep this function as `skipToMicrobreak (delay = 100)` instead of having 2 functions `skipToMicrobreak` and `scheduleMicrobreak`.

Reply:
No worries. That's how I wrote it initially, but then I thought you might prefer more explicit semantics. I'll revert to the original approach.